Source/player.cpp

```diff
@@ -417,7 +418,7 @@ void ChangeOffset(Player &player)
 		player.position.offset2 += player.position.velocity;
 	}
 
-	player.position.offset = { player.position.offset2.deltaX >> 8, player.position.offset2.deltaY >> 8 };
+	player.position.offset = DisplacementOf<int8_t> { static_cast<int8_t>(player.position.offset2.deltaX >> 8), static_cast<int8_t>(player.position.offset2.deltaY >> 8) };

```

Comment:
Nit: This is becomming a bit hard to read... well it wasn't nice before either.
```suggestion
	player.position.offset = DisplacementOf<int8_t> {
		static_cast<int8_t>(player.position.offset2.deltaX >> 8),
		static_cast<int8_t>(player.position.offset2.deltaY >> 8)
	};
```

Reply:
Done

Reply:
Thanks, now it fits on my 80 column monitor again :+1: